### ISSUE TITLE
Error when applying plugin

### ISSUE THREAD

**Issue by vykintazo** [File5/intellij-idea-guidesigner-plugin#1]
Hello,
when I am apply this plugin and try to build, I get this error:
```
An exception occurred applying plugin request [id: 'io.github.file5.guidesigner', version: '1.0.0']
> Failed to apply plugin 'io.github.file5.guidesigner'.
   > Could not get unknown property 'implementation' for configuration container of type org.gradle.api.internal.artifacts.configurations.DefaultConfigurationContainer.
```
I am using IntelliJ IDEA 2021.2.2 and Gradle 7.2 with minimal configuration (just 'application' plugin)

**Comment by File5** [comment]
Hello!
Thank you for the report! I managed to reproduce the problem and fixed it in the version 1.0.1 which is already published.

Feel free to try it out and comment on if it works:
```groovy
plugins {
    id 'application'
    id 'io.github.file5.guidesigner' version '1.0.1'
}
```

I'm describing below what was the problem for the future and for anyone who might be interested.

The problem was connected with the order in which Gradle applies plugins.
So, I usually tested with the following plugin configuration:
```groovy
plugins {
    id 'org.jetbrains.kotlin.jvm' version '1.5.31'
    id 'java'
    id 'io.github.file5.guidesigner' version '1.0.1'
}
```
It looks like the `java` plugin was always applied before `io.github.file5.guidesigner`, probably, because of the presence of `org.jetbrains.kotlin.jvm` plugin. As it turns out when just the `application` plugin is present (which also applies `java`) the order is different from the order of plugins section.

It is related to https://github.com/gradle/gradle/issues/15664 which is already merged and will be included in Gradle 7.3.
But the fix for the `io.github.file5.guidesigner` plugin is done, so it should work on older Gradle versions right now.

**Comment by File5** [comment]
Feel free to open another issue if it doesn't work!
